### PR TITLE
Bump compatibility from 4.10-4.12 to 4.10-4.14

### PR DESCRIPTION
**Pull request by csibbitt**
I have relaxed the version constraint to encompass 4.14, but since it is not yet released, the active parts have been updated to 4.13 only, which is the version I am currently testing on anyways.

### CODE REVIEW

File: build/stf-run-ci/tasks/create_catalog.yml [hunk 1]
```diff
@@ -76,7 +76,7 @@
           dockerfile: |
             # The base image is expected to contain
             # /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
-            FROM registry.redhat.io/openshift4/ose-operator-registry:v4.12
+            FROM registry.redhat.io/openshift4/ose-operator-registry:v4.13
```
Comment (leifmadsen) [inline build/stf-run-ci/tasks/create_catalog.yml:79]: 👍🏻

**Review by leifmadsen — APPROVED** [review]
Did a walk through/search of the code base, and looks like you got everything!

File: build/stf-run-ci/tasks/create_catalog.yml (hunk 1 above)
Comment (csibbitt) [reply build/stf-run-ci/tasks/create_catalog.yml:79]: I'm actually :crossed_fingers: that this change solves the SCC problems, testing now with a local build using `__deploy_from_index_enabled` which I previously noted was broken.

Comment (csibbitt) [reply build/stf-run-ci/tasks/create_catalog.yml:79]: Still broken :-(

**Comment by csibbitt** [comment]
test

**Review by vkmc — CHANGES_REQUESTED** [review]
Code looks good Chris! I would expect to have CI for all the covered OCP versions (or at least, the three EUS versions, OCP 4.10/4.12/4.14). Should we do that as a follow up?

**Comment by leifmadsen** [comment]
> Code looks good Chris! I would expect to have CI for all the covered OCP versions (or at least, the three EUS versions, OCP 4.10/4.12/4.14). Should we do that as a follow up?

I think the issue is that there aren't published 4.14 artifacts in all locations yet, so we'll need to do another round of this closer to the import period. The CVP jobs etc all exist for 4.14 so internal artifacts should exist.

**Comment by csibbitt** [comment]
> Code looks good Chris! I would expect to have CI for all the covered OCP versions (or at least, the three EUS versions, OCP 4.10/4.12/4.14). Should we do that as a follow up?

`registry.redhat.io/openshift4/ose-operator-registry:v4.14` does not exist yet, so yes, we'll have to follow up later.

**Comment by csibbitt** [comment]
This passed testing on 4.10 and 4.12, so merging to continue upstream experimentation

**Pull request merged**